public_html/wp-content/plugins/pattern-creator/src/components/block-editor/index.js

```diff
@@ -73,7 +75,10 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 					contentRef={ mergedRefs }
 					name="editor-canvas"
 				>
-					<BlockList className="pattern-block-editor__block-list wp-site-blocks" />
+					<BlockList
+						className="pattern-block-editor__block-list wp-site-blocks"
+						__experimentalLayout={ layout }
```

Comment:
Passing the layout setting like this is solving a few issues I was seeing when I was passing the old object we copied from Gutenberg, so that's good. 👍